components/log-viewer-webui/server/src/routes/query.js

```diff
@@ -31,29 +85,21 @@ const routes = async (fastify, options) => {
         );
 
         if (null === streamMetadata) {
-            const extractResult = await fastify.dbManager.submitAndWaitForExtractStreamJob({
+            streamMetadata = await extractStreamAndGetMetadata({
+                fastify: fastify,
                 jobType: extractJobType,
                 logEventIdx: sanitizedLogEventIdx,
                 streamId: streamId,
-                targetUncompressedSize: settings.StreamTargetUncompressedSize,
+                resp: resp,
```

Comment:
There's a lint warning (not error) that this should be above `streamId` to maintain alphabetical ordering.